intel_npu_acceleration_library/nn/module.py

```diff
@@ -111,6 +111,7 @@ def __init__(self) -> None:
         self._nn_factory_cache: MutableMapping[str, NNFactory] = {}
         self._npu_inference = False
         self.npu_top_level_module = True
+        self.profile = False
```

Comment:
Can this be part of the module constructor (possibly with a default false value)? so it is easier for people to use it

Reply:
Made this change